packages/lodestar/src/chain/chain.ts

```diff
@@ -49,9 +49,8 @@ export class BeaconChain extends EventEmitter implements IBeaconChain {
   private logger: ILogger;
   private opts: IChainOptions;
   
-  public constructor(opts: Partial<IChainOptions>, {config, db, eth1, opPool, logger}: IBeaconChainModules) {
+  public constructor(opts: IChainOptions, {config, db, eth1, opPool, logger}: IBeaconChainModules) {
```

Comment:
why did you remove partial?

Reply:
resolved via chat. Saving adding Partial<Options> to all constructors as a future PR.